uniffi_aries_vcx/demo/app/src/main/java/org/hyperledger/ariesvcx/ScanScreen.kt

```diff
@@ -66,9 +74,13 @@ fun ScanScreen(connection: Connection, profileHolder: ProfileHolder) {
                         )
                         connection.sendRequest(
                             profileHolder,
-                            "https://google.com",
+                            "$BASE_RELAY_ENDPOINT/send_user_message/${walletConfig.walletKey}",
```

Comment:
using walletKey as the unique ID is very suspicious; wallet key is meant to be a unique password for your device wallet. I'd recommend we just use something else hardcoded as the ID. e.g. `demo-user-1`

Reply:
Oh no I dont know what I was thinking, This is supposed to be a user id, right?